cmd/dump.go

```diff
@@ -56,33 +57,63 @@ var dumpCmd = &cobra.Command{
 
 		fmt.Println("> Fetching policies")
 		wantedPolicies , _ := cmd.Flags().GetStringSlice("policies")
+		wantedAPIs , _ := cmd.Flags().GetStringSlice("apis")
+
 
 		policies := []objects.Policy{}
+		apis := []objects.DBApiDefinition{}
 		var errPoliciesFetch error
+		var errApisFetch error
+
+
+		if len(wantedAPIs) > 0 {
```

Comment:
I think it's safe to remove this check, the iteration should be enough and nothing happens if it's empty.

Reply:
solved